sdcm/nemesis.py

```diff
@@ -1559,6 +1559,10 @@ def disrupt_network_reject_inter_node_communication(self):
         textual_matching_rule, matching_rule = self._iptables_randomly_get_random_matching_rule()
         textual_pkt_action, pkt_action = self._iptables_randomly_get_disrupting_target()
         wait_time = random.choice([10, 60, 120, 300, 500])
+
+        self.monitoring_set.nodes[0].silence_alert(alert_name="YCSBTooManyErrors", duration=wait_time + 500)
+        self.monitoring_set.nodes[0].silence_alert(alert_name="YCSBTooManyVerifyErrors", duration=wait_time + 500)
```

Comment:
why +500?

Reply:
I'm still trying it, we need extra time since the it counts error per minute, and it takes a some time for the alert to disappear. 

I'm still testing it